code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -257,12 +257,13 @@
 		if(I_HURT)
 			var/dealt_damage = harm_intent_damage
 			var/harm_verb = response_harm
-			if(ishuman(M) && M.species)
-				var/datum/unarmed_attack/attack = M.get_unarmed_attack(src)
-				dealt_damage = attack.damage <= dealt_damage ? dealt_damage : attack.damage
-				harm_verb = pick(attack.attack_verb)
-				if(attack.sharp || attack.edge)
-					adjustBleedTicks(dealt_damage)
+			if(ishuman(M))
+				var/decl/natural_attack/attack = M.get_unarmed_attack(src)
```

Comment:
wots happening here?
simple animal is getting victim's unarmed attack?.. and pretending to attack with it?.. but `get_unarmed_attack` doesn't take args?..

Reply:
`M` is the person attacking not the victim. Not sure why it's passing an arg though.